bitnami/nginx/templates/deployment.yaml

```diff
@@ -90,7 +90,8 @@ spec:
             - -ec
             - |
               [[ -f "/opt/bitnami/scripts/git/entrypoint.sh" ]] && source "/opt/bitnami/scripts/git/entrypoint.sh"
-              git clone {{ .Values.cloneStaticSiteFromGit.repository }} --branch {{ .Values.cloneStaticSiteFromGit.branch }} /app
+              git clone {{ .Values.cloneStaticSiteFromGit.repository }} --branch {{ .Values.cloneStaticSiteFromGit.branch }} /tmp/app
+             [[ "$?" -eq 0 ]] && rm -rf /app/* && mv /tmp/repo/* /app/
```

Comment:
You are cloning the repository into `/tmp/app` but then moving `/tmp/repo/*`, could you please take a look into it?

Reply:
@migruiz4 I just copied over the logic from https://github.com/bitnami/charts/pull/7970/files#diff-029bd0a1b3834af7937704d71e07aaeb2b92d9d43713401bc95186bb7f3da9d6R66. It's the same init container steps, which failed similar before, that's why I'm using this code, as it seems to work fine for the apache chart.
What's your worry ?

Reply:
Replying in a new thread, as Github UI won't let me suggest changes on a comment marked as 'Outdated'.